source/_components/sensor.fritzbox_callmonitor.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "FRITZ!Box"
+description: "Instructions how to integrate a phone call monitor for AVM FRITZ!Box routers into Home Assistant."
+date: 2016-08-13 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Sensor
```

Comment:
Can you change this to `System Monitor`

I actually don't like that category either but better then Sensor (it used to be the default but I've been cleaning it up)